code/game/objects/items/weapons/storage/nitrilebox.dm

```diff
@@ -0,0 +1,77 @@
+/obj/item/nitrilebox
+	name = "nitrile gloves box"
+	icon = 'icons/obj/storage.dmi'
+	icon_state = "nitrilebox"
+	item_state = "sheet-metal"
+	drop_sound = 'sound/items/drop/cardboardbox.ogg'
+	pickup_sound = 'sound/items/pickup/cardboardbox.ogg'
+	throwforce = 1
+	w_class = 5
+	throw_speed = 3
+	throw_range = 7
+	layer = OBJ_LAYER - 0.1
+	var/amount = 50					//How much paper is in the bin.  Gloves, actually, but this code is copied from the paper bin code.
+
+/obj/item/nitrilebox/MouseDrop(mob/user as mob)
```

Comment:
```suggestion
/obj/item/nitrilebox/MouseDrop(mob/user)
```

`as mob` is redundant here, because `mob/user` already does this. Also in other places.